bots/tests-scan

```diff
@@ -225,7 +224,7 @@ def scan_for_pull_tasks(api, update, policy):
             # For unmarked and untested status, user must be in whitelist
             # Not this only applies to this specific commit. A new status
             # will apply if the user pushes a new commit.
-            if login not in whitelist:
+            if login not in api.whitelist:
```

Comment:
I think it was at some point, but it shouldn't be in this pr either way, definitely not in this commit.